docs/migrate.to.v1.md

```diff
@@ -0,0 +1,93 @@
+# Migrating from 0.x.x to 1.x.x
+
+There are several significant changes from 0.x.x to 1.x.x:
+* Breaking: Protocol-Common is now published as an ESModule library.
+* Breaking: Minimum node version supported is node 16.
+* Breaking: The underlying logging mechanism has been swapped out for Pino logger.
```

Comment:
Might make sense to actually specify which package has been removed, too

Reply:
Good idea.